modules/accounting/assets/src/admin/components/invoice/InvoiceCreate.vue

```diff
@@ -752,4 +755,87 @@ export default {
             font-size: 1.2em;
         }
     }
+
+    table.wperp-create-invoice-table {
```

Comment:
No need to add a new class. Use the default class and modify that using selector according to your need.